modules/apps/adaptive-media/adaptive-media-image-impl/src/main/java/com/liferay/adaptive/media/image/internal/processor/AMImageProcessorImpl.java

```diff
@@ -44,7 +43,7 @@
  */
 @Component(
 	property = "model.class.name=com.liferay.portal.kernel.repository.model.FileVersion",
-	service = {AMImageProcessor.class, AMProcessor.class}
+	service = AMImageProcessor.class
 )
```

Comment:
Hey @RenanUmV, there is a serviceTracker that tracker `AMImageProcessor` here `AMMessageListener`. So, this fix will break the logic there. Please try to find another way the get rid of these multi services.

Thx